schema/appState.json

```diff
@@ -0,0 +1,390 @@
+                "message": {
+                    "type": "string"
+                }
+            },
+            "required": [],
+            "additionalProperties": false
+        },
+
+        "OdinSolution": {
+            "comment": "This is not really the solution, but the arguments used to generated the solution!",
+            "type": "object",
+            "properties": {
+                "parameterValues": {
+                    "$ref": "#/definitions/OdinParameterValues"
+                },
+                "endTime": {
+                    "type": "number"
+                }
+            }
+        },
+
+        "OdinParameterValues": {
+            "type": "object",
+            "additionalProperties": {
+                "type": "number"
+            }
+        },
+
+        "WodinError": {
+            "type": "object",
+            "properties": {
+                "error": {
+                    "type": "string"
+                },
+                "detail": {
+                    "type": ["null", "string"]
+                }
+            },
+            "required": ["error", "detail"],
+            "additionalProperties": false
+        },
+
+        "SensitivityState": {
+            "paramSettings": {
+                "$ref": "#/definitions/SensitivityParameterSettings"
+            },
+            "batch": {
+                "oneOf": [
+                    { "type": "null" },
+                    { "$ref": "#/definitions/Batch" }
+                ]
+            },
+            "sensitivityUpdateRequired": {
+                "type": "boolean"
+            }
+        },
+
+        "SensitivityParameterSettings": {
+            "parameterToVary": {
+                "type": ["null", "string"]
+            },
+            "scaleType": {
+                "$ref": "#/definitions/SensitivityScaleType"
+            },
+            "variationType": {
+                "$ref": "#/definitions/SensitivityVariationType"
+            },
+            "variationPercentage": {
+                "type": "number"
+            },
+            "rangeFrom": {
+                "type": "number"
+            },
+            "rangeTo": {
+                "type": "number"
+            },
+            "numberOfRuns": {
+                "type": "number"
+            }
+        },
+
+        "SensitivityScaleType": {
+            "type": "string",
+            "enum": ["Arithmetic", "Logarithmic"]
+        },
+
+        "SensitivityVariationType": {
+            "type": "string",
+            "enum": ["Percentage", "Range"]
+        },
+
+        "Batch": {
+            "type": "object",
+            "properties": {
+                "pars": {
+                    "$ref": "#/definitions/BatchPars"
+                },
+                "endTime": {
+                    "type": "number"
+                }
+            },
+            "required": ["pars", "endTime"],
+            "additionalProperties": false
+        },
+
+        "BatchPars": {
```

Comment:
This matches the BatchPars odin type? I don't think we'll need to persist that as we always generate it (in a getter) when we need it.